_documentation/user/flags/jumping.md

```diff
@@ -5,4 +5,17 @@ type: Good
 summary: Tank can jump. Use Tab key. Can't steer in the air.
 ---
 
-...
+The Jumping (JP) flag allows the player to jump from the ground or any object that is touching the bottom side of the tank. An exception are objects with the attributes `drivethrough` or `passable`, where the result is the tank passing through the object.
+
+A lot of maps utilize the default jumping option to already allow the effect of Jumping without the tanks being required to hold the specific flag for it. The Jumping flag holds meaning and effect on maps that do not utilize said option.
+
+## Tactics
+
+Jumping is potent on maps filled with objects, as it allows for more spatial maneuvering. It can be used for either pursuit or escaping purposes, as a player can open up new paths by jumping on or over objects.
+
+On pursuit, it can assist in taking aerial shortcuts to reach the target destinations faster, or ensure a better battle position. While escaping, it can help rabbits move out of hunters' line of fire in the rabbit hunt game mode; it can assist team flag runners to progress further towards their own base while being shot at in the capture the flag game mode; or it can allow tanks to run from opponents holding dangerous close proximity flags, such as [Shock Wave](../shock-wave/).
```

Comment:
how does jumping help escape SW? Since SW has a 3-dimensional kill radius, it's the worst flag to jump from.

Reply:
@jwmelto you are automatically assuming that there are no obstacles around you. Utilizing 'parkour' is an extremely common practice to avoid hitting dead ends when escaping or to swiftly escape the enemy with SW that is not on the same floor/building/height as you on maps such as Urban Jungle.

Reply:
Be careful about assuming what I'm assuming :trollface: 

My experience is that jumping produces slower forward motion than not jumping. When "running" (the verb in the clause under consideration), the idea is to get away as fast as possible. If you have time to jump high enough that SW can't hit you, it probably wasn't that close in the first place. Of course, given that we are talking about the jumping flag, and the SW won't be able to jump after you, there is a case where you can eventually jump out of SW reach. I just wouldn't consider it a primary case. Then again, I don't play a lot of maps that provide the Jumping flag.

Reply:
Jumping does produce slower motion and also less movement flexibility while in the air, but is necessary at times on maps filled with objects to force through a dead end, either jumping over it or onto a higher platform while simultaneously running away from the SW user. In such situations, the player escaping usually tries to rico kill the opponent while, as said, jumping is at times necessary.

Though you are completely right, I messed up here as I somehow forgot we are talking about the JP flag and switched to talking about jumping in general... the Urban Jungle I named doesn't have any JP flags as tanks can jump there already on default. While the point of escaping SW remains (perhaps being even stronger now, as the opponent can't follow you to higher grounds), thanks for the heads-up, ha.